audit/projects/k8s-staging-addon-manager/services/enabled.txt

```diff
@@ -1,7 +1,6 @@
 NAME                              TITLE
 cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
-containeranalysis.googleapis.com  Container Analysis API
```

Comment:
This is the other half of https://github.com/kubernetes/k8s.io/issues/1963#issuecomment-916309207 (the latter half of projects were picked up by https://github.com/kubernetes/k8s.io/pull/2689)